app/assets/stylesheets/admin.scss

```diff
@@ -193,23 +193,21 @@ select {
     align-items: center;
     font-size: 20px;
 
-    .records-per-page {
-      padding: 0 15px;
-      color: $white-color;
-      width: 66px;
-      height: 40px;
-      background-color: $blue-filter-color;
-      border-radius: 10px;
-      border: none;
-    }
-
-    .label-records-per-page  {
+    label {
       margin-left: 5px;
       color: $blue-filter-color;
       font-size: 20px;
     }
 
     input[type=number] {
+      padding-right: 6px !important;
+      color: $white-color;
+      width: 50px !important;
+      height: 40px;
+      background-color: $blue-filter-color;
+      border-radius: 10px;
+      border: none !important;
```

Comment:
is it possible to assign a rule without !important

Reply:
no)